CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 5.1.10 (2022-01-26)
```

Comment:
```suggestion
# 5.1.10 (2022-01-27)
```

Reply:
fixed in commit [eb73d38 - "update changelog per John's suggestions"](https://github.com/blackbaud/skyux-ag-grid/pull/195/commits/eb73d3881c40d8cb015c9edd95e6662f06578685)